moveit_ros/planning_interface/test/python_move_group_ns.py

```diff
@@ -94,7 +94,6 @@ def test_validation(self):
         plan3 = self.plan(current)
         self.assertTrue(self.group.execute(plan3))
 
-
```

Comment:
According to PEP8, this double linebreak was intended/required.

Reply:
Thanks!